PIM/UserInterface/02a_List.md

```diff
@@ -0,0 +1,1419 @@
+# List
+
+*PIM > Products > Tab LIST*
+
+![Products](../../Assets/Screenshots/PIM/Products/List/Products.png "[Products]")
+
+- *View*   
+  Click the drop-down list to select the view. All created views are displayed in the drop-down list. Click the ![Points](../../Assets/Icons/Points01.png "[Points]") (Points) button to the right of the *View* drop-down list to display the context menu and create or edit a view. For detailed information, see [Create a view](../Operation/04_ManageView.md#create-a-view).
+
+  - View context menu   
+    Click the ![Points](../../Assets/Icons/Points01.png "[Points]") (Points) button to the right of the *View* drop-down list to display the context menu. The following menu entries are available:
+
+    - ![Create](../../Assets/Icons/Plus06.png "[Create]") create   
+      Click this entry to create a view. The *Create view* window is displayed.
+
+    - ![Rename](../../Assets/Icons/Edit02.png "[Rename]") rename    
+      Click this entry to rename the selected view. The *Rename view* window is displayed. This menu entry is only displayed when a view is selected.
+
+    - ![Reset](../../Assets/Icons/Reset.png "[Reset]") reset    
+      Click this entry to reset the view to the selected views settings. This menu entry is only displayed when a view is selected and any changes are made to the views settings.
```

Comment:
Diese Option kommt bei mir nicht vor. Refresh button oben links schon.
Text undeutlich für mich, da nicht nachvollziehbar, was genau passiert. Muss man Save nicht drucken, nachdem man Änderungen in einem View vorgenommen hat?
Vorschlag: 
Click this entry to reset the selected view settings / to reset the settings of a selected view. This menu entry is only displayed when a view is selected and changes have been made to the view settings.